---
title: Tools & Workflow
description: Our research project employs a robust methodology that combines computational tools to effectively analyze & assemble genomes from metagenomic data.
illustrationImage:
  - url: /images/homepage_image_part4-min.png
    alt: background image stem cells
nameLink: explore
tabs:
  - name: SemiBin
    link: /resources
    content:
      - paragraph: SemiBin is a tool specifically designed for metagenomic binning, which involves grouping genomic fragments from complex metagenomic samples into individual genomes.
      - paragraph: We developed SemiBin2, an upgraded version of the SemiBin tool, resulting in a significant improvement in the quality of Metagenome-Assembled Genomes (MAGs).
      - paragraph: By incorporating self-supervised contrastive learning, SemiBin2 revolutionizes the metagenomic binning process for both short- and long-read sequencing data.
      - paragraph: The tool makes use of deep siamese neural networks to implement a contrastive learning method in order to accurately classify and separate metagenomic contigs, leading to more precise identification and assembly of individual genomes within complex microbial communities.
      - paragraph: The integration of SemiBin2 in our research workflow has played a vital role in generating higher-quality MAGs, enhancing the accuracy and reliability of our findings.
    workflow:
      - paragraph: Once you have the assembled genomes, you can use SemiBin for binning. Input the assembled genomes into SemiBin along with any additional information required, such as coverage information or other relevant metadata.
      - paragraph: SemiBin utilizes semi-supervised binning methods to classify the metagenomic sequences into taxonomic bins, which will help separate individual genomes from the metagenomic mixture.
  - name: checkM2
    link: /resources
    content:
      - paragraph: SemiBin is a tool specifically designed for metagenomic binning, which involves grouping genomic fragments from complex metagenomic samples into individual genomes.
      - paragraph: We developed SemiBin2, an upgraded version of the SemiBin tool, resulting in a significant improvement in the quality of Metagenome-Assembled Genomes (MAGs).
      - paragraph: By incorporating self-supervised contrastive learning, SemiBin2 revolutionizes the metagenomic binning process for both short- and long-read sequencing data.
      - paragraph: The tool makes use of deep siamese neural networks to implement a contrastive learning method in order to accurately classify and separate metagenomic contigs, leading to more precise identification and assembly of individual genomes within complex microbial communities.
      - paragraph: The integration of SemiBin2 in our research workflow has played a vital role in generating higher-quality MAGs, enhancing the accuracy and reliability of our findings.
    workflow:
      - paragraph: Once you have the assembled genomes, you can use SemiBin for binning. Input the assembled genomes into SemiBin along with any additional information required, such as coverage information or other relevant metadata.
      - paragraph: SemiBin utilizes semi-supervised binning methods to classify the metagenomic sequences into taxonomic bins, which will help separate individual genomes from the metagenomic mixture.
  - name: GUNC
    link: /resources
    content:
      - paragraph: SemiBin is a tool specifically designed for metagenomic binning, which involves grouping genomic fragments from complex metagenomic samples into individual genomes.
      - paragraph: We developed SemiBin2, an upgraded version of the SemiBin tool, resulting in a significant improvement in the quality of Metagenome-Assembled Genomes (MAGs).
      - paragraph: By incorporating self-supervised contrastive learning, SemiBin2 revolutionizes the metagenomic binning process for both short- and long-read sequencing data.
      - paragraph: The tool makes use of deep siamese neural networks to implement a contrastive learning method in order to accurately classify and separate metagenomic contigs, leading to more precise identification and assembly of individual genomes within complex microbial communities.
      - paragraph: The integration of SemiBin2 in our research workflow has played a vital role in generating higher-quality MAGs, enhancing the accuracy and reliability of our findings.
    workflow:
      - paragraph: Once you have the assembled genomes, you can use SemiBin for binning. Input the assembled genomes into SemiBin along with any additional information required, such as coverage information or other relevant metadata.
      - paragraph: SemiBin utilizes semi-supervised binning methods to classify the metagenomic sequences into taxonomic bins, which will help separate individual genomes from the metagenomic mixture.
  - name: tRNAscan-SE
    link: /resources
    content:
      - paragraph: SemiBin is a tool specifically designed for metagenomic binning, which involves grouping genomic fragments from complex metagenomic samples into individual genomes.
      - paragraph: We developed SemiBin2, an upgraded version of the SemiBin tool, resulting in a significant improvement in the quality of Metagenome-Assembled Genomes (MAGs).
      - paragraph: By incorporating self-supervised contrastive learning, SemiBin2 revolutionizes the metagenomic binning process for both short- and long-read sequencing data.
      - paragraph: The tool makes use of deep siamese neural networks to implement a contrastive learning method in order to accurately classify and separate metagenomic contigs, leading to more precise identification and assembly of individual genomes within complex microbial communities.
      - paragraph: The integration of SemiBin2 in our research workflow has played a vital role in generating higher-quality MAGs, enhancing the accuracy and reliability of our findings.
    workflow:
      - paragraph: Once you have the assembled genomes, you can use SemiBin for binning. Input the assembled genomes into SemiBin along with any additional information required, such as coverage information or other relevant metadata.
      - paragraph: SemiBin utilizes semi-supervised binning methods to classify the metagenomic sequences into taxonomic bins, which will help separate individual genomes from the metagenomic mixture.
  - name: barrnap
    link: /resources
    content:
      - paragraph: SemiBin is a tool specifically designed for metagenomic binning, which involves grouping genomic fragments from complex metagenomic samples into individual genomes.
      - paragraph: We developed SemiBin2, an upgraded version of the SemiBin tool, resulting in a significant improvement in the quality of Metagenome-Assembled Genomes (MAGs).
      - paragraph: By incorporating self-supervised contrastive learning, SemiBin2 revolutionizes the metagenomic binning process for both short- and long-read sequencing data.
      - paragraph: The tool makes use of deep siamese neural networks to implement a contrastive learning method in order to accurately classify and separate metagenomic contigs, leading to more precise identification and assembly of individual genomes within complex microbial communities.
      - paragraph: The integration of SemiBin2 in our research workflow has played a vital role in generating higher-quality MAGs, enhancing the accuracy and reliability of our findings.
    workflow:
      - paragraph: Once you have the assembled genomes, you can use SemiBin for binning. Input the assembled genomes into SemiBin along with any additional information required, such as coverage information or other relevant metadata.
      - paragraph: SemiBin utilizes semi-supervised binning methods to classify the metagenomic sequences into taxonomic bins, which will help separate individual genomes from the metagenomic mixture.
---

Homepage section4 content
